### DISCUSSION TITLE
OF changed their API

### DISCUSSION THREAD

**Discussion by pmarbibi12** [DIGITALCRIMINALS/OnlyFans#1044] [General]
doesnt seem like the scraper will work anymore.

**Comment by ImSoDan** [comment]
I've seen someone else mention that there is now a "DRM" option when videos are uploaded. It seems that the videos are streamed in a different method for those videos. When I inspect the page while the video is playing there are hundreds of video chunks in mp4 format being downloaded constantly. The videos themselves are just black if you download them through the inspector.

**Reply by pmarbibi12** [reply]
Yeah, I used to manually download the mp4 files by inspecting the html and looking for the source address. That's no longer available. sad sad day
